packages/admin/admin/src/form/Field.tsx

```diff
@@ -6,7 +6,12 @@ import { FormattedMessage } from "react-intl";
 import { FieldContainer, FieldContainerProps } from "./FieldContainer";
 import { useFinalFormContext } from "./FinalFormContextProvider";
 
-const requiredValidator = (value: any) => (value ? undefined : <FormattedMessage id="comet.form.required" defaultMessage="Required" />);
+const requiredValidator = (value: any) => {
+    if (value === undefined || value === null || value === false || value === "") {
```

Comment:
@thomasdax98 Should we trim the string? Otherwise, `" "` bypasses the required check, which I'm not sure is wanted.

Reply:
https://github.com/vivid-planet/comet/pull/2521